tests/drivers/flash/boards/stm32f746g_disco.overlay

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2021 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+       chosen {
+               zephyr,flash-controller = &flash;
+       };
+};
+
+&quadspi {
+       qspi-nor-flash@0 {
+               partitions {
+                       /delete-node/ partition@a0000;
+               };
+       };
+};
+
+&flash0 {
+        partitions {
+               compatible = "fixed-partitions";
+               #address-cells = <1>;
+               #size-cells = <1>;
+
+                /* Set 2KB of storage at the end of 1MB flash */
+                storage_partition: partition@ff800 {
+                        label = "storage";
+                        reg = <0x000FF800 0x00000800>;
```

Comment:
usually written in lower case    
       `               reg = <0x000ff800 0x00000800>;`